frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -975,15 +976,15 @@ function ReaderFooter:textOptionTitles(option)
         percentage = symbol_prefix[symbol].percentage
             and T(_("Progress percentage (%1)"), symbol_prefix[symbol].percentage) or _("Progress percentage"),
         book_time_to_read = symbol_prefix[symbol].book_time_to_read
-            and T(_("Book time to read (%1)"),symbol_prefix[symbol].book_time_to_read) or _("Book time to read"),
-        chapter_time_to_read = T(_("Chapter time to read (%1)"), symbol_prefix[symbol].chapter_time_to_read),
+            and T(_("Time left to read book (%1)"),symbol_prefix[symbol].book_time_to_read) or _("Time left to read book"),
+        chapter_time_to_read = T(_("Time left to read chapter (%1)"), symbol_prefix[symbol].chapter_time_to_read),
         frontlight = T(_("Frontlight level (%1)"), symbol_prefix[symbol].frontlight),
-        frontlight_warmth = T(_("Frontlight warmth (%1)"), symbol_prefix[symbol].frontlight_warmth),
+        frontlight_warmth = T(_("Frontlight warmth level (%1)"), symbol_prefix[symbol].frontlight_warmth),
```

Comment:
Ugly, it looks and sounds much better as:
```suggestion
        frontlight_warmth = T(_("Frontlight warmth (%1)"), symbol_prefix[symbol].frontlight_warmth),
```